README.md

```diff
@@ -0,0 +1,290 @@
+# Composable rules
+
+Imagine you need to write rules to modify some data based on a growing set of
+a growing set of business logic.
```

Comment:
Is the phrase a growing set of repeated twice?

Reply:
oh yes it is, good catch.